cloudify_hostpool/storage/base.py

```diff
@@ -97,3 +97,21 @@ def update_host(self, global_id, new_values, old_values=None):
         :rtype `dict`
 
         """
+
+    @abc.abstractproperty
+    def has_initialised_storage(self):
+        # Later we will remove this flag from the interface and change
+        # the way we pass this information.
```

Comment:
For now we're leaving this method solving tge multiple storage population problem.